### ISSUE TITLE
Posts image should have a full url, not relative path

### ISSUE THREAD

**Issue by raelmax** [raelmax/city#3]
When we fetch a post some images don't load because are saved with a relative path. :(

**Comment by snahor** [comment]
This should be taken care of by the feed producer, IMHO.

BTW, there's no link to the original "article". I've modified the template to include it. If you want I can open a PR for it. Take a look https://github.com/raelmax/city/compare/master...snahor:add-feed-link

**Comment by raelmax** [comment]
I agree with you,  but i think that we can easily fix this on our side instead of wait for feed producer fix this problem. :)

Send me your PR for title link, I'll thankful to accept it. Thanks!
